Frameworks/include/CoreGraphics/D2DWrapper.h

```diff
@@ -36,6 +36,10 @@ inline D2D_POINT_2F _CGPointToD2D_F(CGPoint point) {
     return { point.x, point.y };
 }
 
+inline CGPoint _D2DPointToCGPoint(D2D_POINT_2F point) {
+    return CGPointMake(point.x, point.y);
```

Comment:
I've been following the pattern of d2d to cg using the make functions.

---
In reply to: [90974360](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r90974360) [](ancestors = 90974360)